core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceBindingBuilder.java

```diff
@@ -158,6 +160,17 @@ public AnnotatedServiceBindingBuilder useBlockingTaskExecutor(boolean useBlockin
         return this;
     }
 
+    /**
+     * Sets the query parameter delimiter. It is disabled by default.
+     *
+     * <p>Note that this delimiter works only when the resolve target class type is collection and the number
+     * of values of the query parameter is one.</p>
+     */
+    public AnnotatedServiceBindingBuilder useQueryDelimiter(String delimiter) {
```

Comment:
I think we can omit the `use` prefix.
```suggestion
    public AnnotatedServiceBindingBuilder queryDelimiter(String delimiter) {
```